src/main/java/com/dezorganizacja/bridge/presenter/MainPresenter.java

```diff
@@ -2,8 +2,11 @@
 
 import com.dezorganizacja.bridge.domain.Message;
 
-public class MainPresenter {
+import java.util.Observable;
+
+public class MainPresenter extends Observable {
     private final Message message;
+    private String state;
```

Comment:
`String`, nie `String`, nie musimy się tym zajmować tutaj. Stan gry będzie pochodził z jakichś wnętrzności (aka. paczki `domain`). Na potrzeby tego PR może zostać.

@wTendera @wolowiec pewnie będziecie potrzebowali zaprojektować to jakoś sensowniej po stronie logiki gry.